index.js

```diff
@@ -58,17 +58,218 @@ const renderMarkdown = async (path) => {
 	return marked(raw);
 }
 
+app.get('/misc/General-FAQ', async (req, res) => {
+    res.redirect('/getting-started/faq');
```

Comment:
Should we set these to be perma redirects? It's better for SEO IIRC

```suggestion
    res.redirect(301, '/getting-started/faq');
```

Reply:
done, thanks for catching